06week/higherOrder.js

```diff
@@ -2,18 +2,45 @@
 
 const assert = require('assert');
 
-function forEach(arr, callback) {
-  // Your code here
-}
-
-function map(arr, callback) {
-  // Your code here
-}
-
-function filter(arr, callback) {
-  // Your code here
-}
 
+// function forEach(arr, callback){
+const forEach = (inputArray, inputFunction) => {
+  const anotherArr = [];
+  for (let i = 0; i < inputArray.length; i++){
+    anotherArr.push(inputFunction(inputArray[i]));
```

Comment:
Pushing to an array that isn't returned, then logging the length. This isn't what `forEach()` does.